src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -205,6 +205,23 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
 
         return viewableSessions;
     }
+    
+    /**
+     * checks if there is any email sent from the course
+     * @param courseId
+     * @param userEmail
+     * @return true if there is any email sent from the course.
+     */
+    public boolean isAnyEmailSentForTheCourse(String courseId) {
```

Comment:
is**Any**Email actually sounds misleading; it could well include closing/closed/unpublished email. You may want to rename the method to accurately represent what it does.